package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@blackbird/utils",
+  "version": "0.0.0-development",
```

Comment:
I'm picturing publishing the first version as `0.1.0`. Then for non-breaking changes, we can release `0.1.x`, and for a breaking change bump to `0.2.0`. I believe that then if we're using `^` versions when using this as a dependency, that'll work as far as semantic versioning (eg `"@blackbird/utils": "^0.1.0"` won't match `0.2.0`)?